saleor/settings.py

```diff
@@ -647,3 +647,16 @@ def get_host():
 
 assert hasattr(schema_printer, "_print_object")
 schema_printer._print_object = patched_print_object
+
+# Settings for populatedb command
+DEFAULT_USERS_PASSWORD = "password"
+
+POPULATE_SUPER_USER_PASSWORD = os.environ.get(
+    "POPULATE_SUPER_USER_PASSWORD", DEFAULT_USERS_PASSWORD
```

Comment:
Is it expected that this will change our default password for `admin@example.com` from `admin` to `password`?
Can't we still use old good `admin` here?

Reply:
We shouldn't change the "admin" password. It is used for example in our demo where we automatically prefill it in the frontend, so changing it here will require us to also fix the demo.

Also, I think we only need at most two env variables:
1) for superuser - "admin"
2) for everyone else - "password"